tests/run.sh

```diff
@@ -1,30 +1,31 @@
 #!/bin/bash
 set -x -e
 RUN_S3=""
+MENDER_BRANCH=$(cd .. && python extra/release_tool.py --version-of mender)
```

Comment:
You should not need to call python directly anymore after your other fix.

It should also not be necessary to `cd`, since the script will detect its own location and work from there.